src/core/services/gesture/gesture.js

```diff
@@ -318,6 +331,19 @@ function MdGesture($$MdGestureHandler, $$rAF, $timeout) {
       }
     });
 
+  function getTouchAction() {
+    var testEl = angular.element('<div>')[0];
+    var vendorPrefixes = ['', 'webkit', 'Moz', 'MS', 'ms', 'o'];
```

Comment:
Well it's probably not slowing down too much, but no other prefixes ,going back to the oldest browser version, weren't mentioned.